### PR TITLE
Website cleanup in preparation for SemanticDB v3

### PR DESCRIPTION
**Pull request by xeno-by**
As discussed in the "SemanticDB 3.0: Website Changes" proposal.

### CODE REVIEW

File: readme/LandingPage.scalatex [hunk 1]
```diff
@@ -237,17 +212,24 @@
   @sect{Tutorial}
     @p
       To learn more about practical aspects of using scalameta, take a look at
-      our tutorial that is based on a workshop given by Ólafur Pall Geirsson at Scala World 2016:
+      our tutorial written by Ólafur Pall Geirsson:
```
Comment (olafurpg) [inline readme/LandingPage.scalatex]: remove "written by ..."?

File: readme/LandingPage.scalatex [hunk 2]
```diff
@@ -237,17 +212,24 @@
...
       @lnk("Scalafix", "https://scalacenter.github.io/scalafix/") is a
-      code rewriting tool developed at the @lnk("Scala Center", "https://scala.epfl.ch/").
+      rewrite and linting tool for Scala developed at the @lnk("Scala Center", "https://scala.epfl.ch/").
       Scalafix rewrites use the Scalameta to automate migration between different library and compiler versions.
 
+    @p
+      @b
+        Language server protocol.
+      @lnk("Metals", "https://geirsson.com/assets/metals/") is an experimental project
+      that use Scalameta-based tools to implement
```
Comment (olafurpg) [inline readme/LandingPage.scalatex]: implement **the**

File: readme/tutorial/Semantic.scalatex [hunk 3]
```diff
@@ -9,89 +9,28 @@
...
       The storage format used for the SemanticDB is defined using
-      @lnk("Protocol Buffers", "https://www.google.ch/search?q=protocol+buffers&oq=protocol+buffer&aqs=chrome.0.0j69i61j69i57j69i61j0l2.6856j0j7&sourceid=chrome&ie=UTF-8"),
+      @lnk("Protocol Buffers", "https://developers.google.com/protocol-buffers/"),
       or "protobuf" for short.  The SemanticDB protobuf schema is small, at
-      the time of this writing it is ~50 lines of code.  The full schema is
+      the time of this writing it is ~60 lines of code.  The full schema is
       available
       @lnk("here",
-           "https://github.com/scalameta/scalameta/blob/master/langmeta/shared/src/main/protobuf/semanticdb.proto").
+           "https://github.com/scalameta/scalameta/blob/master/semanticdb/semanticdb2/semanticdb2.proto").
```
Comment (olafurpg) [inline readme/tutorial/Semantic.scalatex:29]: We should really try to stop moving this file around, I think this is like the 5th location we keep it at.

**Review by olafurpg — APPROVED** [review]
LGTM 👍

File: readme/LandingPage.scalatex (hunk 1 above)
Comment (xeno-by) [reply readme/LandingPage.scalatex]: Done.

File: readme/LandingPage.scalatex (hunk 2 above)
Comment (xeno-by) [reply readme/LandingPage.scalatex]: Done.

File: readme/tutorial/Semantic.scalatex (hunk 3 above)
Comment (xeno-by) [reply readme/tutorial/Semantic.scalatex:29]: 😅

**Pull request merged**